client-side/src/components/MyButton.scss

```diff
@@ -0,0 +1,22 @@
+@import url('https://fonts.googleapis.com/css2?family=Nanum+Gothic&display=swap');
+
+.myButton{
```

Comment:
don't use important in scss files

Reply:
so what should i do?

Reply:
you need to use more specific hierarchic, so it will override the existing CSS, for example instead of using only .myButton you can add there mui button classes also, so it will be overridden.
you can try to find in google how can you override mui design

Reply:
I found a solution, thanks